server/src/main/java/org/elasticsearch/action/search/TransportSearchAction.java

```diff
@@ -147,7 +150,8 @@ public TransportSearchAction(
         ActionFilters actionFilters,
         IndexNameExpressionResolver indexNameExpressionResolver,
         NamedWriteableRegistry namedWriteableRegistry,
-        ExecutorSelector executorSelector
+        ExecutorSelector executorSelector,
+        QueriesUsageService queryUsageService
```

Comment:
When I look at this in IntelliJ, I see a warning "Class 'QueriesUsageService' is not exported from module 'org.elasticsearch.server'".